app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_flag, first_file, second_file = command.split()
+    if command_flag == "mv" and len(command.split()) == 3:
+        directory, new_file = os.path.split(second_file)
```

Comment:
don't split it twice command.split()
create a separated variable and use it on demand